docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,43 @@
+|------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Kotlin DataFrame](https://github.com/Kotlin/dataframe)                      | - Data collection <br/> - Data cleaning and processing                                                                   | - Operations for creating, sorting, and cleaning data frames, feature engineering, and more  <br/> - Processing of structured data  <br/> - Support for `CSV`, `JSON`, and other input formats <br/> - Reading from SQL databases                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| [Kotlin for Apache Spark](https://github.com/JetBrains/kotlin-spark-api)     | - Data collection <br/> - Data cleaning and processing <br/> - Data exploration and visualization <br/> - Model building | - Layer of compatibility between [Apache Spark](https://spark.apache.org/) and Kotlin <br/> - Apache Spark data transformation operations in Kotlin-idiomatic code <br/> - Simple usage of Kotlin features, such as data classes and lambda expressions, in curly braces or method reference                                                                                                                                                                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                                                              
```

Comment:
The order should be following:
Kotlin DataFrame
Kandy
KotlinDL
Multik
Kotlin for Apache Spark
Letsplot

Reply:
There are our libraries, supported and maintained in JetBrains